client/rest/test/routes/routeUtils_spec.js

```diff
@@ -606,95 +606,86 @@ describe('route utils', () => {
 				});
 			});
 		});
+
 		describe('send data', () => {
-			describe('send data', () => {
-				const sendDataTest = (sender, assertResponse) => {
-					// Arrange
-					const routeContext = { numNextCalls: 0 };
-					const next = () => { ++routeContext.numNextCalls; };
-					const data = [];
-					const headers = [];
-
-					const res = {
-						setHeader: (name, value) => {
-							headers.push({ name, value });
-						},
-						write: d => {
-							data.push(d);
-						},
-						end: () => {}
-					};
-					routeContext.responses = {};
-					routeContext.responses.headers = headers;
-					routeContext.responses.data = data;
-
-					// Act: send the entity
-					sender(res, next);
-
-					assertResponse(routeContext.responses);
-				};
-				const send = ({
-					data, mimeType, fileName, text, download
-				}, type, assertResponse) => {
-					sendDataTest((res, next) =>
-						routeUtils.createSender(type).sendData(res, next)(data, mimeType, fileName, text, download), assertResponse);
+			const sendDataTest = (sender, assertResponse) => {
+				// Arrange
+				const routeContext = { numNextCalls: 0 };
+				const next = () => { ++routeContext.numNextCalls; };
+				const data = [];
+				const headers = [];
+
+				const res = {
+					setHeader: (name, value) => {
+						headers.push({ name, value });
+					},
+					write: writeData => {
+						data.push(writeData);
+					},
+					end: () => {}
 				};
+				routeContext.responses = { headers, data };
 
-				it('send data', () => {
-					// Act:
-					send({ data: Buffer.from([0, 1, 2]), mimeType: 'imnage/png' }, 'foo', response => {
-						// Arrange:
-						const contentTypeValue = response.headers.find(header => 'content-type' === header.name)?.value;
-						const contentDispositionValue = response.headers.find(header => 'Content-Disposition' === header.name)?.value;
-						// Assert:
-						expect(response.data[0]).to.deep.equal(Buffer.from([0, 1, 2]));
-						expect(contentDispositionValue).to.equal('inline;');
-						expect(contentTypeValue).to.equal('imnage/png');
-					});
+				// Act: send the entity
+				sender(res, next);
+
+				expect(routeContext.numNextCalls).to.equal(1);
+				assertResponse(routeContext.responses);
+			};
+			const send = ({
+				data, mimeType, fileName, text, download
+			}, type, assertResponse) => {
+				sendDataTest((res, next) =>
+					routeUtils.createSender(type).sendData(res, next)(data, mimeType, fileName, text, download), assertResponse);
+			};
+
+			const testHeader = (response, dataBuffer, disposition, mimeType, text) => {
```

Comment:
testHeader is a noun so code reads like it is a header vs a header test function.
i'd suggest rename this to assertHeader.